src/common/redis_module/ray_redis_module.cc

```diff
@@ -1,14 +1,26 @@
-#include "redismodule.h"
 #include <string.h>
 
-#include "redis_string.h"
-
 #include "common_protocol.h"
 #include "format/common_generated.h"
 #include "ray/gcs/format/gcs_generated.h"
 #include "ray/id.h"
+#include "redis_string.h"
+#include "redismodule.h"
```

Comment:
What does the style guide say about this? `redismodule.h` is not really our code.

Reply:
Well, it is a file physically in the ray source tree, in that sense we should consider it as our project's source.